### DISCUSSION TITLE
Merge People

### DISCUSSION THREAD

**Discussion by jstride** [ente-io/ente#3536] [Enhancements]
With the ML face recognition, it can give duplicates for people (think kids growing up). It would be helpful to be able to merge 'multiple people' into one.

**Comment by laurenspriem** [comment]
This is already possible on mobile. After naming a person and going into that group of photos, you will be prompted to review suggestions, which is the easiest and semi-automatic way of merging them. You can also merge two groups manually by assigning them the same name.

**Reply by heliostatic** [reply]
Will this make it to Desktop, too?

**Reply by mnvr** [reply]
Yes!

**Reply by 1silvertiger** [reply]
How to do you name a person and view their photos on Android?

**Reply by laurenspriem** [reply]
> How to do you name a person and view their photos on Android?

Have you turned on ML? If not please first go to `Settings > Advanced > Machine Learning`, turn it on and then wait for it to finish indexing. Once that is done people should start showing up on the search screen. Once you tap on a certain person a banner should pop up asking you to name the person.

**Reply by 1silvertiger** [reply]
@laurenspriem Thanks for the tip. I did not have ML turned on on my Android device. Now I can see people and faces in Android, but it seems to have synced the people from my desktop, including a duplicate of a person.

**Reply by laurenspriem** [reply]
That is intentional. ML includes E2EE sync between devices. Referring back to this original issue, you can easily merge duplicates. Please see [here](https://help.ente.io/photos/faq/machine-learning#can-i-merge-or-de-merge-persons-recognized-by-the-app) how.

**Reply by 1silvertiger** [reply]
Thank you! I had already named the duplicate, so I removed the person label from the duplicate and then followed the instructions at the link.
